build_tools/python/e2e_test_framework/models/torch_models.py

```diff
@@ -0,0 +1,52 @@
+## Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""Defines PyTorch models."""
+
+from e2e_test_framework import unique_ids
+from e2e_test_framework.definitions import common_definitions
+
+# Implementations of the models listed below can be found in `https://github.com/iree-org/iree-samples/tree/main/iree-torch/importer`.
+# We import the PyTorch models offline and make the .mlir available here for benchmarking.
+# If the mlir artifacts need to be updated, please run [update_torch_models.sh](https://github.com/iree-org/iree-samples/blob/main/iree-torch/importer/update_torch_models.sh)
+# Then update the `source_url` below with the new paths.
+
+# `ClipTextModel` encodes text into an embedding.
+#
+# Used in Stable Diffusion to convert a text prompt into an embedding for input to the `Unet2d` model.
+#
+# Converted from https://huggingface.co/docs/transformers/model_doc/clip#transformers.CLIPTextModel
+MODEL_CLIP_TEXT_SEQLEN64_FP32_TORCH = common_definitions.Model(
+    id=unique_ids.MODEL_CLIP_TEXT_SEQLEN64_FP32_TORCH,
+    name="ClipTextSeqLen64PT",
+    tags=["fp32", "seqlen64"],
+    source_type=common_definitions.ModelSourceType.EXPORTED_LINALG_MLIR,
+    source_url=
+    "https://storage.googleapis.com/iree-model-artifacts/pytorch/torch_models_20230307.103_1678163233/SD_CLIP_TEXT_MODEL_SEQLEN64/linalg.mlir",
+    entry_function="forward",
+    input_types=["1x77xi64", "1x77xi64"])
```

Comment:
Did we ever add correctness testing to our benchmark suites? If so, I'd expect some sort of "expected outputs" file (for iree-run-module, or iree-run-trace?) to also be hosted in the model artifacts bucket here. (We should be testing correctness for everything we benchmark, but we don't need to benchmark everything that we test for correctness)

Reply:
+1 for correctness testing. The model implementations in `iree-samples` save the input and output arrays in .npy format. I believe there is an open item to add correctness testing here. @pzread